src/main/resources/validation_errors.properties

```
@@ -1,3 +1,4 @@
 size.min = size must be greater or equal to {min}
 null_or_not_blank.error = must be null or not blank
 date_interval.error = endDate must be after startDate
+files.invalid_image = invalid image format
```

```suggestion
files.invalid_image = invalid image type (png, jpg or jpeg)
```
Something more informative would be nice

---

any decision on this?